src/3d/framegraph/qgsambientocclusionrenderview.h

```diff
@@ -0,0 +1,91 @@
+/***************************************************************************
+  qgsambientocclusionrenderview.h
+  --------------------------------------
+  Date                 : June 2024
+  Copyright            : (C) 2024 by Benoit De Mezzo and (C) 2020 by Belgacem Nedjima
+  Email                : benoit dot de dot mezzo at oslandia dot com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSAMBIENTOCCLUSIONRENDERVIEW_H
+#define QGSAMBIENTOCCLUSIONRENDERVIEW_H
+
+#include "qgsabstractrenderview.h"
+#include "qgsambientocclusionrenderentity.h"
+#include "qgsambientocclusionblurentity.h"
```

Comment:
these includes should go to .cpp and header should only have forward declaration

Reply:
done